jenkins/buildroot-builder.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 env
 
-# Build 
-./configs/frags/build ${arch}
+arch=${1}
+variant=${2:-base}
+
+# Build
+./configs/frags/build ${arch} ${variant}
```

Comment:
The arg order is important here, we want options from $arch to override options from $variant (especially when variant == base) so this should be:
`./configs/frags/build ${variant} ${arch} `